reindexer/complete_reindex/src/test_complete_reindex.py

```diff
@@ -70,6 +70,25 @@ def _create_event(shard_id, current_version, desired_version):
     }
 
 
+def test_should_retry():
+    err_good = ClientError({
+        'Error': {
+            'Code': 'ProvisionedThroughputExceededException',
+            'Message': 'oops'
+        }
+    },'testing')
+
+    err_bad = ClientError({
+        'Error': {
+            'Code': 'Bangarang!',
+            'Message': 'oops'
+        }
+    },'testing')
+
+    assert should_retry(err_good) == True
+    assert should_retry(err_bad) == False
```

Comment:
And a test that `should_retry(AssertionError("foo"))` is False?

Reply:
Yup, good plan.